include/tvm/relay/attrs/vision.h

```diff
@@ -14,7 +14,7 @@ namespace relay {
 
 /*! \brief Attributes for upsampling operator */
 struct UpSamplingAttrs : public tvm::AttrsNode<UpSamplingAttrs> {
-  double scale;
+  int scale;
```

Comment:
we should still keep this as double, as there can be fractional scale size, but try to make sure the inference deal wth it

Reply:
Ref. https://github.com/dmlc/tvm/blob/0247b97198b0fecf82e6ca44d2f34bc8dfc3540f/nnvm/include/nnvm/top/nn.h#L378
It's an int in NNVM. Please confirm to use double here.

The test case here gets tricky with ```int32((float32(100)*2.000000f))``` in side shape !!